### PR TITLE
Update documentation for partitioned heat conduction

### PR DESCRIPTION
**Pull request by BenjaminRodenberg**
* Add information about higher-order time stepping schemes
* Update information about running
* Link to thesis of Niklas Vinnitchenko and Tobias Eppacher

### CODE REVIEW

File: partitioned-heat-conduction/README.md [hunk 1]
```diff
@@ -39,28 +39,37 @@ You can either couple a solver with itself or different solvers with each other.
 
 ## Running the simulation
 
-You can find the corresponding `run.sh` script for running the case in the folders corresponding to the solver you want to use.
+You can find the corresponding `run.sh` script for running the case in the folders corresponding to the participant you want to use.
 
-For choosing whether you want to run the Dirichlet-kind and a Neumann-kind participant, please provide the following commandline input:
+For choosing whether you want to run the Dirichlet-kind and a Neumann-kind participant, please navigate to the corresponding folder (e.g. `dirichlet-fenics` or `neumann-openfoam`) and use the `run.sh` script. For running the case, open two terminals and run:
```
Comment (MakisH) [inline partitioned-heat-conduction/README.md]: Too much duplication here, I think you can simplify (GitHub does not let me suggest a multi-line edit here):

```
You can find the corresponding `run.sh` script for running the case in the folders corresponding to the participant you want to use:
```

File: perpendicular-flap/solid-fenics/requirements.txt
Comment (MakisH) [inline perpendicular-flap/solid-fenics/requirements.txt:1]: These files are unrelated. Maybe rebase the branch to develop?

Comment (BenjaminRodenberg) [reply perpendicular-flap/solid-fenics/requirements.txt:1]: Removed them.

File: partitioned-heat-conduction/README.md (hunk 1 above)
Comment (BenjaminRodenberg) [reply partitioned-heat-conduction/README.md]: Done. Thanks!

**Review by MakisH — APPROVED** [review]
